### ISSUE TITLE
在自己的数据集上训练

### ISSUE THREAD

**Issue by yafeim** [siriusdemon/Build-Your-Own-Face-Model#18]
博主你好！感谢分享文章和代码。

使用了你的代码后有一些疑问想请教一下：

基本情况：
- 网络结构：resnet + arcface loss
- 数据集：自己制作的数据集
- 参数：batch size 试过了64和128，其他的超参数是用的默认的，训练了100个epoch。
- 结果：第一个epoch 的training loss是18，到第10个epoch时变成了14，然后再往后就不再下降了。并且validation loss一直在19左右浮动，没有下降过。

想请教的问题：1. 要做什么调整才能让 training loss收敛呢？2. validation loss为什么不下降？

期待你的答复，感谢！

**Comment by siriusdemon** [comment]
有很多原因。我只能给建议：在你自己的数据集上训练之前，你先用你的超参数在我所使用的数据上训练。看看结果
